meeting-minutes/2025/2025-06-18.md

```diff
@@ -0,0 +1,145 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+- Justin & Omar open the meeting.
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 8 or more of the 14 voting members (including the officers).
+
+| First Name | Last Name   | Company                                                     | Role(s)       | Present |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|:--------|
+| Adrian     | Diglio      | Microsoft                                                   | Member        | No      |
+| Art        | Middlekauff | Flexera                                                     | Voting Member | No      |
+| David      | Kemp        | National Security Agency                                    | Member        | No      |
+| Denny      | Page        | Individual                                                  | Voting Member | No      |
```

Comment:
OQC seems to be down currently, but I believe I remember Denny being present but he just had to leave early. Will check to confirm later.